makefile

```diff
@@ -423,10 +440,14 @@ else ifeq ($(call TRY_COMPILE, $(FIO_TLS_TEST_BEARSSL_EXT), "-lbearssl"), 0)
   $(info * Detected the BearSSL library, setting HAVE_BEARSSL)
 	FLAGS:=$(FLAGS) HAVE_BEARSSL
```

Comment:
Please add a TODO comment, to add `FIO_TLS_FOUND` once BearSSL is supported (just so we remember what we did).

Reply:
Done.